app/src/components/radioGroup/Radio.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {
+  Radio as AriaRadio,
+  type RadioProps as AriaRadioProps,
+} from "react-aria-components";
+import { css } from "@emotion/react";
+
+import { classNames } from "@phoenix/components";
+import { StylableProps } from "@phoenix/components/types";
+
+const baseRadioCSS = css(`
+    padding: var(--ac-global-dimension-size-50) var(--ac-global-dimension-size-100);
+    border: 1px solid var(--ac-global-input-field-border-color);
+    background-color: var(--ac-global-input-field-background-color);
+    color: var(--ac-global-text-color-900);
+    &[data-selected="true"] {
```

Comment:
I like how in the ToggleButtonGroup the selected one extends the color into the border - it makes the component look more "designed" for lack of a better word.

<img width="675" alt="Screenshot 2025-01-22 at 8 46 12 AM" src="https://github.com/user-attachments/assets/e16d8827-6a95-4850-a349-c7ecab4a64a9" />